package.json

```diff
@@ -261,7 +262,7 @@
     "end-of-stream": "^1.4.4",
     "eth-block-tracker": "^6.0.0",
     "eth-ens-namehash": "^2.0.8",
-    "eth-json-rpc-filters": "^4.2.1",
+    "eth-json-rpc-filters": "^5.0.0",
```

Comment:
I confirmed the only breaking change from v4.2.1 to 5 is minimum node version bump to 14.